### PR TITLE
[219] - [FE] Implement remove avatar on user settings

### PR DESCRIPTION
**Pull request by rogeliojohnoliverio**
## Issue Link
- https://app.asana.com/0/1203011167276287/1203370347690219/f

## Definition of Done
- [x] Implemented the remove photo button 
- [x] Replaces the current photo with a default photo

## Notes
- None

## Pre-condition
- Login
- Open User settings
- Click the remove button

## Expected Output
- Should be able to remove the current photo to the default one

## Screenshots/Recordings

https://user-images.githubusercontent.com/110364637/202118556-f326ab82-adbb-4408-a542-97e967dfcfc9.mp4

### CODE REVIEW

File: client/src/components/organisms/SettingsModal/index.tsx [hunk 1]
```diff
@@ -94,6 +95,22 @@ const SettingsModal = ({ close }: { close: (value: boolean) => void }) => {
     )
   }
 
+  const removeImage = (): void => {
+    toast.promise(
+      dispatch(removePhoto(id))
+        .unwrap()
+        .then((_: any) => {
+          dispatch(hydrateUserState())
```
Comment (abduljalilpalala) [inline client/src/components/organisms/SettingsModal/index.tsx]:
```suggestion
        .then(() => {
          dispatch(hydrateUserState())
```

File: client/src/hooks/taskMethods.ts [hunk 2]
```diff
@@ -167,9 +169,14 @@ export const useTaskMethods = (projectID: number) => {
...
-      setIsTaskLoading(false)
-    })
+    dispatch(getTask(task_id))
+      .unwrap()
+      .then((_) => {
+        setIsTaskLoading(false)
+      })
+      .catch((_) => {
+        router.push('/not-found')
+      })
```
Comment (abduljalilpalala) [inline client/src/hooks/taskMethods.ts]:
```suggestion
      .then(() => {
        setIsTaskLoading(false)
      })
      .catch(() => {
        router.push('/not-found')
      })
```

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]
Please check comments.

**Review by jsvelte — APPROVED** [review]
# Approved 💯

File: client/src/hooks/taskMethods.ts [hunk 3]
```diff
@@ -167,9 +169,14 @@ export const useTaskMethods = (projectID: number) => {
...
-      setIsTaskLoading(false)
-    })
+    dispatch(getTask(task_id))
+      .unwrap()
+      .then(() => {
+        setIsTaskLoading(false)
+      })
+      .catch(() => {
+        router.push('/not-found')
+      })
```
Comment (abduljalilpalala) [inline client/src/hooks/taskMethods.ts]:
```suggestion
      .then(() => setIsTaskLoading(false))
      .catch(() => router.push('/not-found'))
```

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✔️

**Pull request merged**